tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -591,11 +550,13 @@ private static SavedModelBundle fromHandle(
           .getValueList()
           .forEach(
               node -> {
-                graph.registerInitOp(graph.operationOrThrow(node));
+                graph.registerInitializer(graph.operationOrThrow(node), false);
               });
     }
 
-    return new SavedModelBundle(graph, session, metaGraphDef, functions);
+    return new SavedModelBundle(graph, session, metaGraphDef, functions.entrySet().stream()
```

Comment:
Why'd this transformation move out of the constructor?

Reply:
Bah, just a little preference of keeping constructors as simple as possible and moving any logic to the caller (so if that logic throws an exception, it won't be thrown from the constructor). I really don't mind reverting this, don't even recalled changing it :)